pkg/coscheduling/coscheduling.go

```diff
@@ -228,6 +228,21 @@ func (cs *Coscheduling) Permit(ctx context.Context, state *framework.CycleState,
 		waitTime = 0
 	}
 
+	// We will also request to move the sibling pods back to activeQ.
+	if podsToMove := cs.pgMgr.SiblingPods(pod); len(podsToMove) != 0 {
```

Comment:
Another suggestion: Maybe this whole functionality should be in `pgMgr.ActivateSiblings(state)` and inline `SiblingPods` inside that function?